taginfo.json

```diff
@@ -0,0 +1,41 @@
+{
+    "data_format": 1,
+    "data_url": "https://raw.githubusercontent.com/Rudloff/openvegemap/master/taginfo.json",
+    "project": {
+        "name": "OpenVegeMap",
+        "description": "Find vegetarian and vegan restaurants in your city",
+        "project_url": "https://github.com/Rudloff/openvegemap",
+        "doc_url": "https://play.google.com/store/apps/details?id=pro.rudloff.openvegemap",
```

Comment:
Would it be possible to use [this page](https://wiki.openstreetmap.org/wiki/OpenVegeMap) instead?

Reply:
Done, it's what I had first.